src/prebid.js

```diff
@@ -375,6 +375,16 @@ $$PREBID_GLOBAL$$.removeAdUnit = function (adUnitCode) {
   }
 };
 
+/**
+ * Remove all adUnits from the $$PREBID_GLOBAL$$ configuration
+ * @alias module:pbjs.removeAllAdUnits
+ */
+$$PREBID_GLOBAL$$.removeAllAdUnits = function () {
```

Comment:
hmmm should we make this `removeAdUnits([adUnitCodeArray])` ? seems more flexible.